hyperspy/models/eelsmodel.py

```diff
@@ -976,3 +978,83 @@ def resume_auto_fine_structure_width(self, update=True):
                 self.resolve_fine_structure()
         else:
             warnings.warn("Not suspended, nothing to resume.")
+
+    def estimate_and_set_coreloss_edge_onset(
```

Comment:
Wouldn't it be good to add a ``mask`` kwarg to mask positions where the edge is not present?